pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -236,8 +236,9 @@ def __init__(self, **kwargs):
 
     def do_build_flow(self, **kwargs):
         flow = linear_flow.Flow('grpc-codegen')
-        flow.add(protoc_tasks.ProtoCodeGenTask('ProtoGen', inject=kwargs),
-                 protoc_tasks.GrpcCodeGenTask('GrpcCodegen', inject=kwargs))
+        kwargs.update({'packman_flags': ['--experimental_alt_java']})
+        flow.add(protoc_tasks.GrpcPackmanTask('Packman', inject=kwargs),
+                 package_tasks.GrpcPackageDirTask('PackageDir', inject=kwargs))
```

Comment:
Good catch. I think you're right; the package dir task appears only to be needed if there is a subsequent task in the pipeline that requires the argument `package_dir`.